java-security/src/main/java/com/sap/cloud/security/servlet/HybridTokenFactory.java

```diff
@@ -43,14 +44,18 @@
 	 * @return the new token instance
 	 */
 	public Token create(String jwtToken) {
+		try {
+			Objects.requireNonNull(jwtToken, "Requires encoded jwtToken to create a Token instance.");
+			DecodedJwt decodedJwt = Base64JwtDecoder.getInstance().decode(removeBearer(jwtToken));
 
-		Objects.requireNonNull(jwtToken, "Requires encoded jwtToken to create a Token instance.");
-		DecodedJwt decodedJwt = Base64JwtDecoder.getInstance().decode(removeBearer(jwtToken));
-
-		if (isXsuaaToken(decodedJwt)) {
-			return new XsuaaToken(decodedJwt).withScopeConverter(getOrCreateScopeConverter());
+			if (isXsuaaToken(decodedJwt)) {
+				return new XsuaaToken(decodedJwt).withScopeConverter(getOrCreateScopeConverter());
+			}
+			return new SapIdToken(decodedJwt);
+		} catch (JsonParsingException e) {
+			LOGGER.error("Issue with Jwt parsing. Authorization header: {} - {}", jwtToken.substring(0, 20), e.getMessage());
```

Comment:
this log isn't required, when you raise a new exception with that message. 
sth like:
````
String message = "Issue with Jwt parsing. Authorization header: {} - {}", jwtToken.substring(0, 20), e.getMessage()
throw new JsonParsingException(message, e);
````